### DISCUSSION TITLE
How to make custom collections to be referenced on the internal links?

### DISCUSSION THREAD

**Discussion by kouohhashi** [payloadcms/payload#2770] [Q&A]
Hi,

I added a couple of collections and it seems works fine.
But when I create internal links, UI only show default collections like Page, Post, User, and Media.

I could not figure our where I should put information about my custom collections.

Could someone give me instruction?

Thanks in advance.

**Comment by DanRibbens** [comment]
Are you asking about the RichText field's link feature?

It should show all collections in your project as options by default. We recently added ways to remove collections from the select, but that isn't what you're asking for.

I think this is a bug with the select we're using for internal links. I'll create a new issue from this discussion.
